### DISCUSSION TITLE
Printing from within Safe Exam Browser

### DISCUSSION THREAD

**Discussion by airsay** [SafeExamBrowser/seb-win-refactoring#468] [Q&A] [answered]
I am coming here from https://sourceforge.net/p/seb/discussion/844844/thread/b5ef9875/. This question was asked when SEB was in v2.2/2.2.1. Given that SEB is currently iv v3.4.0, I wanted to ask if there is any way (easy or not) to enable printing on SEB. I understand that printing is not recommended with SEB but I have a very specific use case were printing is required. TIA

**Comment by dbuechel** [comment, marked as answer]
Starting with SEB 3.4.0, there is a new configuration option to control printing of web content (see Browser > Browser Security in the configuration tool). So yes, in general you should be able to print content of the integrated web browser when this setting is enabled.

**Reply by airsay** [reply]
@dbuechel Thank you for pointing out that update. Much appreciated
